src/Sign.Cli/AzureCredentialType.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE.txt file in the project root for more information.
+
+namespace Sign.Cli
+{
+    internal static class AzureCredentialType
+    {
+        public const string Environment = "environment";
```

Comment:
Should we add `managed-identity` now?  @clairernovotny, thoughts?

Reply:
Might be better to do this in separate PR's?